notebooks/ts_generic_example.ipynb

```diff
@@ -296,6 +297,7 @@
     "            Layer(classical_dense_layer=ClassicalDenseLayer(hidden_size=1, bias=False)),\n",
     "        ],\n",
     "        num_epochs=5,\n",
+    "        k_fold=1,\n",
```

Comment:
if you are also adding k_fold here. Please then also upgrade the grpc client version from the dependencies.

Reply:
done